foundations/html_css/flexbox/flexbox-axes.md

```diff
@@ -54,4 +54,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
\ No newline at end of file
+*   [This]([url](https://flexbox.malven.co/)) cheatsheet has some useful references to flex and its properties.
```

Comment:
Great idea to add this resource! Can we make the link text more descriptive instead of just saying "this"?

```suggestion
*   [This flexbox visual cheatsheet]([url](https://flexbox.malven.co/)) has some useful references to flex and its properties.
```